src/metafile.c

```diff
@@ -873,9 +873,15 @@ gdip_metafile_clone (GpMetafile *metafile, GpMetafile **clonedmetafile)
 	if (!mf)
 		return OutOfMemory;
 
+	memcpy (&mf->base.image_format, &metafile->base.image_format, sizeof (GUID));
```

Comment:
why is copy base.image_format special, don't we need all other fields too?

Reply:
No other fields are used. Do you think we should just copy M.D.->image?

Reply:
I see e.g. num_of_frames used a couple times in the codebase, not sure why you think only image_format is used :)

I assume we need to do a deep copy of all the fields, otherwise it wouldn't be a true clone?

Reply:
> I see e.g. num_of_frames used a couple times in the codebase, not sure why you think only image_format is used :)

I believe that for each of these, this is hidden behind the `ImageTypeBitmap` case.

But maybe it's just a good idea to do deep copy everything

Reply:
Yeah. Let me know if you want to do this as part of this PR or separately, I'm fine with merging the current state since it's already better than what was there before :)